docs/src/app/(content)/layout.tsx

```diff
@@ -3,7 +3,8 @@ import * as React from 'react';
 import { AppBar } from 'docs/src/components/AppBar';
 import { Navigation } from 'docs/src/components/Navigation';
 import routes from 'docs/data/pages';
-import '../../styles/style.css';
+import 'docs/src/styles.css';
+import 'docs/src/styles/style.css';
```

Comment:
This is temporary just to support the current setup in place, `'docs/src/styles/style.css'` will go away eventually